08-evaluation-1/PRESENTATION.md

```diff
@@ -0,0 +1,118 @@
+---
+marp: true
+---
+
+<!--
+theme: gaia
+size: 16:9
+paginate: true
+author: L. Delafontaine and H. Louis, with the help of ChatGPT
+title: 'HEIG-VD DAI Course - Evaluation 1'
+description: 'Evaluation 1 for the DAI course at HEIG-VD, Switzerland'
+url: https://heig-vd-dai-course.github.io/heig-vd-dai-course/08-evaluation-1/
+footer: '**HEIG-VD** - DAI Course 2023-2024 - CC BY-SA 4.0'
+style: |
+    :root {
+        --color-background: #fff;
+        --color-foreground: #333;
+        --color-highlight: #f96;
+        --color-dimmed: #888;
+        --color-headings: #7d8ca3;
+    }
+    blockquote {
+        font-style: italic;
+    }
+    table {
+        width: 100%;
+    }
+    th:first-child {
+        width: 15%;
+    }
+    h1, h2, h3, h4, h5, h6 {
+        color: var(--color-headings);
+    }
+    h2, h3, h4, h5, h6 {
+        font-size: 1.5rem;
+    }
+    h1 a:link, h2 a:link, h3 a:link, h4 a:link, h5 a:link, h6 a:link {
+        text-decoration: none;
+    }
+    section:not([class=lead]) > p, blockquote {
+        text-align: justify;
+    }
+headingDivider: 4
+-->
+
+[web]: https://heig-vd-dai-course.github.io/heig-vd-dai-course/08-evaluation-1/
+[pdf]:
+  https://heig-vd-dai-course.github.io/heig-vd-dai-course/08-evaluation-1/08-evaluation-1-presentation.pdf
+[license]:
+  https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/LICENSE.md
+[illustration]:
+  https://images.unsplash.com/photo-1604134967494-8a9ed3adea0d?fit=crop&h=720
+
+# Evaluation 1
+
+<!--
+_class: lead
+_paginate: false
+-->
+
+<https://github.com/heig-vd-dai-course>
+
+[Web][web] · [PDF][pdf]
+
+<small>L. Delafontaine and H. Louis, with the help of ChatGPT.</small>
+
+<small>This work is licensed under the [CC BY-SA 4.0][license] license.</small>
+
+![bg opacity:0.1][illustration]
+
+## Evaluation
+
+In order to validate this part of the course, you will have to pass an
+evaluation. The questions will be in French and you can answer them in French or
+English.
+
+The evaluation will only be **a theoretical part of 60 minutes**.
+
+The evaluation will be done on the computer with the help the online evaluation
+platform as used in the evaluation 1.
+
+You will have to be connected to the
+[VPN](https://intranet.heig-vd.ch/services/informatique/poste-de-travail/reseau/vpn/Pages/vpn.aspx).
+It is **mandatory**.
```

Comment:
It seems to me that if we're connected to the school network, the VPN is no longer necessary.